src/main/java/com/fdmgroup/backend_streamhub/livechat/service/MessageListener.java

```diff
@@ -0,0 +1,21 @@
+package com.fdmgroup.backend_streamhub.livechat.service;
+
+import com.fdmgroup.backend_streamhub.livechat.models.Message;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MessageListener {
+//    @Autowired
+//    SimpMessagingTemplate template;
```

Comment:
Remove comments and add later in SCRUM-71 when needed again

Reply:
done